rhombus/rhombus/scribblings/reference/filesystem.scrbl

```diff
@@ -72,3 +72,15 @@
  directory cannot be found.
 
 }
+
+@doc(
+  fun filesystem.list_directory(path :: PathString
+                                  = Path.current_directory(),
+                                ~extend_path: extend = #false) :: List.of(Path)
```

Comment:
Nit: Can you move the result annotation to a new line?

Reply:
No problem